DTS-STN/Alpha-Site
# Feature/#132 experiment component

# Description

[#132 Create an "experiment" component](https://trello.com/c/tAppcybk/132-132-create-an-experiment-component)

![image](https://user-images.githubusercontent.com/31868510/115765940-5a837d80-a375-11eb-8b28-36b867ed1d4a.png)



## Acceptance Criteria

 - The blue tag must be the same as the bottom border.
 - There are 3 data elements on the card: title, phase tag, description.

## Test Instructions

1. git checkout Feature/#132-ExperimentComponent 
2. npm run storybook
3. See Experiment component under atoms.
4. Make sure it matches the Experiment card on the mockups.

## Help Requested

Feedback on how to improve is always welcome.

## Checklist


## Product and Sprint Backlog

[#132 Create an "experiment" component]https://trello.com/c/tAppcybk/132-132-create-an-experiment-component
